packages/discord.js/src/structures/Integration.js

```diff
@@ -185,6 +185,16 @@ class Integration extends Base {
     } else {
       this.application ??= null;
     }
+
+    if ('scopes' in data) {
+      /**
+       * The scopes the application has been authorized for
```

Comment:
It seems everything else uses "this integration" so I think that should be changed for consistency:

```suggestion
       * The scopes this application has been authorized for
```